sample/src/main/res/drawable/ic_edit.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We shouldn't need to add this to both the component module and the sample module. We either need to build it in to the component (which I think is fine for this one) or have it supplied by the consumer. So I think we can remove this one, as we have added it to the component itself.